### PR TITLE
chore: update package.json + lint test-files

### PR DESCRIPTION
**Pull request by felicia-haggqvist**
Part of fixes for JIRA-ticket [WARP-520](https://nmp-jira.atlassian.net/browse/WARP-520)

- Remove `lint:eslint` since it is a duplicate of `lint:check`
- Add `--ext js,cjs,mjs` to `lint:check` and `lint:fix`

### CODE REVIEW

**Comment by felicia-haggqvist** [comment]
@imprashast Do you know why we have both prettier and eslint in the package.json? In the package.json we have this: 
 ```
"format": "prettier --write . --ignore-path .gitignore",
"lint:format": "prettier --check . --ignore-path .gitignore",
"lint": "npm run lint:format && pnpm run lint:eslint",
"lint:fix": "eslint . --fix --ext js,cjs,mjs --ignore-path .gitignore",
"lint:check": "eslint . --ext js,cjs,mjs --ignore-path .gitignore",
```
Should we have both prettier and eslint or just eslint? :)

**Review by BalbinaK — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -32,11 +32,10 @@
     "commit": "cz",
     "dev": "vite",
     "format": "prettier --write . --ignore-path .gitignore",
```
Comment (imprashast) [inline package.json:34]:
```suggestion
    "dev": "vite",
    "lint": "eslint . --fix --ext js,cjs,mjs --ignore-path .gitignore",
    "lint:check": "eslint . --ext js,cjs,mjs --ignore-path .gitignore",
```

I would say, remove format, lint:format, lint:fix as well and change lint to what lint:fix is doing

Comment (BalbinaK) [reply package.json:34]: We are using `lint:check` in the github action (I guess that will change once the husky integration is in place).

Comment (imprashast) [reply package.json:34]: `lint:check` is still unchanged

Comment (BalbinaK) [reply package.json:34]: Sorry, my brain is malfunctioning today. I thought you asked about `lint:check` 😅 It sounds good to have `lint` do the fixing, and `lint:check` do the checking 👍

**Review by imprashast — APPROVED** [review]
Awesome 👍

**Pull request merged**